dynet/cudnn-ops.cu

```diff
@@ -88,35 +89,13 @@ void CudnnConvOp::forward_impl(const Device_GPU& dev, const std::vector<const Te
     }
   }
 
-  // set cudnn descriptors
-  CUDNN_CHECK(cudnnSetTensor4dDescriptor(x_desc_, 
-              CUDNN_TENSOR_NCHW, DataTypeToCudnnType<float>::value,
-              XN, XC, XW, XH));
-  CUDNN_CHECK(cudnnSetTensor4dDescriptor(y_desc_, 
-              CUDNN_TENSOR_NCHW, DataTypeToCudnnType<float>::value,
-              YN, YC, YW, YH));
-#if CUDNN_VERSION_MIN(5, 0, 0)
-  CUDNN_CHECK(cudnnSetFilter4dDescriptor(filter_desc_, 
-              DataTypeToCudnnType<float>::value, CUDNN_TENSOR_NCHW,
-              FYC, FXC, FW, FH));
-#else
-  CUDNN_CHECK(cudnnSetFilter4dDescriptor_v4(filter_desc_, 
-              DataTypeToCudnnType<float>::value, CUDNN_TENSOR_NCHW,
-              FYC, FXC, FW, FH));
-#endif
-#if CUDNN_VERSION_MIN(6, 0, 0)
-  CUDNN_CHECK(cudnnSetConvolution2dDescriptor(conv_desc_,
-              pad_w/2, pad_h/2, stride_[1], stride_[0], 1, 1,
-              CUDNN_CROSS_CORRELATION, CUDNN_DATA_FLOAT));
-#else
-  CUDNN_CHECK(cudnnSetConvolution2dDescriptor(conv_desc_,
-              pad_w/2, pad_h/2, stride_[1], stride_[0], 1, 1,
-              CUDNN_CROSS_CORRELATION));
-#endif
+  //set cudnn descriptors
+  setTensor4dDescriptor(&x_desc_, XN, XC, XW, XH);
```

Comment:
I think it'd be better to be consistent with the use of "descriptor" and "desc". I can see people being confused about which one to use. If there is no strong preference, I'd suggest exactly matching the CUDNN name, minus the `cudnn`.